tock/employees/tests/test_models.py

```diff
@@ -212,3 +208,24 @@ def test_token_is_delete_on_active_is_false(self):
         except Token.DoesNotExist:
             token_after_save = None
         self.assertNotEqual(token_before_save, token_after_save)
+
+    def test_is_18f_employee_false_if_no_org(self):
+        """False if no org or not named 18F"""
+        self.regular_user_userdata.organization = None
+        self.regular_user_userdata.save()
+        self.assertFalse(self.regular_user_userdata.is_18f_employee)
+
+    def test_is_18f_employee_false_if_not_18f(self):
+        """False if org not named 18F"""
+        not_18f = Organization.objects.create(
+            name='not_18f',
+            description='not_18f',
+            active=True
+        )
+        self.regular_user_userdata.organization = not_18f
+        self.regular_user_userdata.save()
+        self.assertFalse(self.regular_user_userdata.is_18f_employee)
+
+    def test_is_18f_employee_true_if_18f(self):
+        """True if org is named 18F"""
+        self.assertTrue(self.regular_user_userdata.is_18f_employee)
```

Comment:
This one doesn't need an org set because it's set in the test setup method?

Reply:
Correct. Up on L65

Reply:
Yaas, but that's not clear from this one line test alone--- :enhance: